platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/RichTextEditor.kt

```diff
@@ -135,9 +138,11 @@ private fun RealEditor(
 
                 applyDefaultStyle()
 
-                // Restore the state of the view with the saved state
-                setHtml(state.internalHtml)
-                setSelection(state.selection.first, state.selection.second)
+                // Set initial HTML and selection, state will be restored automatically in the view
+                if (editableText.isEmpty()) {
+                    setHtml(state.internalHtml)
+                    setSelection(state.selection.first, state.selection.second)
+                }
```

Comment:
Is it still necessary to make this change after https://github.com/matrix-org/matrix-rich-text-editor/pull/865 ?

Reply:
Nope, removing this now and uploading some new changes.